src/jvmMain/kotlin/JvmMain.kt

```diff
@@ -0,0 +1,23 @@
+import komem.litmus.*
+import komem.litmus.barriers.JvmSpinBarrier
+import komem.litmus.testsuite.IRIWVolatile
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+    val runner: LTRunner = JvmThreadRunner
+    val test = IRIWVolatile
+
+    val syncEverySchedule = generateSequence(10) { (it * 1.7).toInt() }.takeWhile { it < 1000 }.toList()
+    println("len = ${syncEverySchedule.size}")
```

Comment:
What is this?
A comment explaining this logic would be helpful.

Reply:
I mean what is `syncEverySchedule` and how it is used?
It is not clear from the code.